frontend/src/metabase/lib/formatting/time.ts

```diff
@@ -67,3 +61,35 @@ export function formatTimeWithUnit(
 
   return m.format(timeFormat);
 }
+
+interface TimeOnlyOptions {
```

Comment:
Wdyt about moving this type to `frontend/src/metabase/lib/formatting/types.ts`?
I think it would be reasonable to call it `TimeOptions` and extend `OptionsType` with it to avoid duplication

Reply:
So, OptionsType extends from TimeOptions? It seems like other small interfaces have more "locality of behavior" (they live near their functions if only used once), but I don't have strong opinion on this.

Reply:
Done.